lib/hyacinth/adapters/external_identifier_adapter/datacite/rest_api/v2/api.rb

```diff
@@ -15,52 +15,70 @@ class Hyacinth::Adapters::ExternalIdentifierAdapter::Datacite::RestApi::V2::Api
                      findable: 'publish',
                      registered: 'hide' }.freeze
 
-  attr_accessor :most_recent_request_body
+  attr_accessor :most_recent_request_body,
+                :most_recent_response
 
   def initialize(datacite_rest_api, basic_auth_user, basic_auth_password)
     @datacite_rest_api = datacite_rest_api
     @basic_auth_user = basic_auth_user
     @basic_auth_password = basic_auth_password
-    @most_recent_request_body = nil
+    @most_recent_request = nil
```

Comment:
this approach might present some thread safety issues - the adapters are shared across the app, if I remember correctly.

Reply:
+1 from me about @barmintor's comment

Reply:
instance variables have been removed.